File: app/src/main/java/com/example/searchablerecyclerview/Animal.java
package com.example.searchablerecyclerview;

public class Animal {
    String name;
    String location;
    String img;

    public Animal() {
    }

    public String getName() {
        return name;
    }

    public String getLocation() {
        return location;
    }

    public String getImage() {
        return "http://192.168.74.212/"+img;
    }
}

File: app/src/main/java/com/example/searchablerecyclerview/kotlin/Animal.kt
package com.example.searchablerecyclerview.kotlin

class Animal {
    var name: String? = null
    var location: String? = null
    var img: String? = null

    constructor(name: String?, location: String?, img: String?) {
        this.name = name
        this.location = location
        this.img = "http://192.168.75.5/"+img
    }
}